ui/components/app/transaction-activity-empty-state/transaction-activity-empty-state.test.tsx

```diff
@@ -1,20 +1,58 @@
 import React from 'react';
-import { screen } from '@testing-library/react';
+import { screen, fireEvent } from '@testing-library/react';
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
+import { InternalAccount } from '@metamask/keyring-internal-api';
+import { EthMethod } from '@metamask/keyring-api';
 import { renderWithProvider } from '../../../../test/jest';
 import mockState from '../../../../test/data/mock-state.json';
 import { ThemeType } from '../../../../shared/constants/preferences';
+import useBridging from '../../../hooks/bridge/useBridging';
 import {
   TransactionActivityEmptyState,
   type TransactionActivityEmptyStateProps,
 } from './transaction-activity-empty-state';
 
+// Mock the useBridging hook
+jest.mock('../../../hooks/bridge/useBridging');
+
+const mockAccount: InternalAccount = {
+  id: 'cf8dace4-9439-4bd4-b3a8-88c821c8fcb3',
+  address: '0x0dcd5d886577d5081b0c52e242ef29e70be3e7bc',
+  type: 'eip155:eoa',
+  options: {},
+  scopes: ['eip155:1'],
+  methods: [
+    'personal_sign',
+    'eth_sign',
+    'eth_signTransaction',
+    'eth_signTypedData_v1',
+    'eth_signTypedData_v3',
+    'eth_signTypedData_v4',
+  ],
+  metadata: {
+    name: 'Account 1',
+    keyring: { type: 'HD Key Tree' },
+    importTime: Date.now(),
+  },
+};
+
 describe('TransactionActivityEmptyState', () => {
   const middleware = [thunk];
+  const mockOpenBridgeExperience = jest.fn();
+  const mockUseBridging = useBridging as jest.MockedFunction<
+    typeof useBridging
+  >;
```

Comment:
Some pretty dirty typing here but not sure of any other approach. Open to suggestions @micaelae